exercises/runLengthEncode/runLengthEncode.js

```diff
@@ -20,11 +20,52 @@
  * @returns {string} A run-length encoded copy of the input string
  */
 
-function runLengthEncode(num) {
-  // This is your job. :)
+function runLengthEncode(string) {
+  let element = string[i]
+  let newArray= []
+  for (let i = 0; i < string.length; i++) {
+    if {
+      element = element[i +1]
+      count++
+    }
+   else {
+     element != 
+   } 
+  }
+}
+
+function stringtoArray(string) {
```

Comment:
Both `string.split('')` and `Array.from(string)` will get you an array containing the letters of a string.